packages/core/src/build/handlers.ts

```diff
@@ -148,23 +152,18 @@ export const buildRawHandlerFunctions = async ({
     (name) => name !== outAppFilename
   );
 
-  const requireErrors = outUserFilenames
-    .map((file) => {
-      try {
-        require(file);
-        return undefined;
-      } catch (err) {
-        return err as Error;
-      }
-    })
-    .filter((err): err is Error => err !== undefined);
+  const importUserFiles = await Promise.allSettled(
+    outUserFilenames.map((filename) => import(filename))
+  );
+
+  const rejecteds = importUserFiles.filter(isRejected);
```

Comment:
This change to allSettled makes sense, though I'd probably call this `rejectedImports`.

Reply:
done